docs/src/tutorials/symmetric_mesh.md

```diff
@@ -0,0 +1,360 @@
+
+For a domain bounded by an outer boundary curve, this background grid is set by indicating the desired
+element size in the $x$ and $y$ directions. To start, we set the background grid for `symmetric_mesh`
+to have elements with side length $0.25$ in each direction
+```julia
+addBackgroundGrid!(spline_project, [0.25, 0.25, 0.0])
+```
+
+## Add the outer boundary
+
+With the background grid size set, we next build the outer boundary for the present mesh project.
+This outer boundary is composed of nine straight line segments and a half circle arc.
+The curves will afterwards be added to the mesh project `symmetric_mesh`
+in counter-clockwise order as required by HOHQMesh.
+```julia
+line1 = newEndPointsLineCurve("symmetry", [-0.05, 2.0, 0.0],
+                                          [-0.05, 0.0, 0.0])
+
+line2 = newEndPointsLineCurve("bottom", [-0.05, 0.0, 0.0],
+                                        [1.0, 0.0, 0.0])
+
+line3 = newEndPointsLineCurve("right", [1.0, 0.0, 0.0],
+                                       [1.0, 0.5, 0.0])
+
+half_circle = newCircularArcCurve("circle",         # curve name
+                                  [1.0, 0.75, 0.0], # circle center
+                                  0.25,             # circle radius
+                                  270.0,            # start angle
+                                  90.0,             # end angle
+                                  "degrees")        # angle units
+
+line4 = newEndPointsLineCurve("right", [1.0, 1.0, 0.0],
+                                       [1.0, 1.5, 0.0])
+
+line5 = newEndPointsLineCurve("bump", [1.0, 1.5, 0.0],
+                                      [0.75, 1.5, 0.0])
+
+line6 = newEndPointsLineCurve("bump", [0.75, 1.5, 0.0],
+                                      [0.75, 1.75, 0.0])
+
+line7 = newEndPointsLineCurve("bump", [0.75, 1.75, 0.0],
+                                      [1.0, 1.75, 0.0])
+
+line8 = newEndPointsLineCurve("right", [1.0, 1.75, 0.0],
+                                       [1.0, 2.0, 0.0])
+
+line9 = newEndPointsLineCurve("top", [1.0, 2.0, 0.0],
+                                     [-0.05, 2.0, 0.0])
+```
+The given boundary names will also be the element boundary names written to the mesh file.
+The only exception is the first boundary curve that is given
+the name `"symmetry"`. The name of this outer boundary curve is a special keyword
+in HOHQMesh that says it is the straight line across which
+a reflection will occur.
+
+!!! tip "Name of the symmetry boundary"
+    As noted above, `"symmetry"` is a keyword for the HOHQMesh generator that
+    prescribes over which line a reflection of the mesh occurs. The Fortran implementation
+    for this keyword in HOHQMesh is not case sensitive. So, within the
+    Julia script one can name this reflection boundary line `"symmetry"` or `"Symmetry"`
+    or `"SYMMETRY"` or even something strange like `"sYmMeTrY"` and the mesh will still
+    generate successfully.
```

Comment:
David and I discussed this name a bit and settled on "symmetry". However, making a specialized variant of the name to avoid accidental misuse is a good idea.